presto-main/src/test/java/io/prestosql/execution/TestSetRoleTask.java

```diff
@@ -87,14 +89,29 @@ public void testSetRole()
         assertSetRole("SET ROLE bar", ImmutableMap.of(CATALOG_NAME, new SelectedRole(SelectedRole.Type.ROLE, Optional.of("bar"))));
     }
 
+    @Test
+    public void testSetRoleInvalidCatalog()
+    {
+        assertPrestoExceptionThrownBy(() -> executeSetRole("invalid", "SET ROLE foo"))
```

Comment:
Can you please add tests for other statements?

Reply:
Done